src/solvers/solveCobraLPCPLEX.m

```diff
@@ -179,6 +179,7 @@
 %conflict refinement is desired in the case that infeasibility is detected
 %by CPLEX.
 if conflictResolve
+    cpxControl.PARALLEL = 1;

```

Comment:
I meant to say actually different solver interfaces, such as Tomlab CPLEX and/or ILOG cplex (my bad).

May we add a test for both interfaces?